package.json

```diff
@@ -11,6 +13,35 @@
     "start": "node ./index.js",
     "test": "run-s lint jest"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/renovatebot/renovate-approve-bot-bitbucket-cloud.git"
+  },
+  "keywords": [
+    "bitbucket",
+    "dependencies",
+    "dependency",
+    "docker",
+    "management",
+    "outdated",
+    "update"
+  ],
+  "author": "Maxime Brunet <insert-email-here>",
```

Comment:
@maxbrunet Can you create a PR suggestion to make this the `author` field how you want it? I don't know how you want to be credited as the original author.